src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1591,38 +1592,45 @@ public async Task<JArray> CreateJArrayForProperties(int typeId, ElementType elem
                 if (getMethodId == 0 || await GetParamCount(getMethodId, token) != 0 || await MethodIsStatic(getMethodId, token))
                     continue;
                 JObject propRet = null;
-                if (attributes.Where(attribute => attribute["name"].Value<string>().Equals(propertyNameStr)).Any())
+                if (attributesCollections
+                    .Any(attributes => attributes
+                    .Where(attribute => attribute["name"].Value<string>().Equals(propertyNameStr)).Any()))
                     continue;
                 if (isAutoExpandable)
                 {
-                    try {
+                    try
+                    {
                         propRet = await InvokeMethod(object_buffer, getMethodId, propertyNameStr, token);
                     }
                     catch (Exception)
                     {
-                        continue;
+                        propRet = GetNotAutoExpandableObject(getMethodId, propertyNameStr);
                     }
                 }
                 else
```

Comment:
after the `if (autoExpandable) {...}` block, you can just check `if (propRet == null) ...`.